STANDARD.md

```diff
@@ -81,6 +81,28 @@ In general, projects should at a minimum have a test task that invokes the
 tests and generates test coverage reports and a CodeCov upload task that
 uploads reports to CodeCov.io.
 
+##### pullRequest policy
+
+Projects should use the `public_restricted` pull request policy if they use any
```

Comment:
I can see an argument for this being `public_restricted` regardless of whether secrets are currently used - mainly to safeguard against future secrets being added (and forgetting to update the policy).

What do you think?

Reply:
Sure! It makes it a bit simple to decide as well.